src/lib/Form/TrashLocationStrategy/TrashLocationStrategy.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\TrashLocationStrategy;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use Symfony\Component\Form\FormInterface;
+
+interface TrashLocationStrategy
```

Comment:
I'm not sure if `TrashLocationStrategy` is the best naming choice. Maybe`TrashLocationOptionProvider` or `TrashLocationOption`?